soccer/afcom/client.go

```
@@ -0,0 +1,134 @@
+	errC := make(chan error, 1)
+	conn, _, err := websocket.DefaultDialer.DialContext(ctx, URL.String(), http.Header(val))
+	if err != nil {
+		errC <- err
+		close(errC)
+		return nil, errC
+	}
+
+	resC := make(chan []soccer.Fixture, 1)
+	go func() {
+		defer func() {
+			conn.WriteMessage(websocket.CloseMessage, websocket.FormatCloseMessage(websocket.CloseNormalClosure, ""))
```

golangci-lint (1.64.8)

Error return value of `conn.WriteMessage` is not checked

(errcheck)